op-e2e/actions/ecotone_fork_test.go

```diff
@@ -132,14 +134,49 @@ func TestEcotoneNetworkUpgradeTransactions(gt *testing.T) {
 
 	// 4788 contract is deployed
 	expected4788Address := crypto.CreateAddress(derive.EIP4788From, 0)
-	code, err := engine.EthClient().CodeAt(context.Background(), expected4788Address, latestBlock.Number())
-	require.NoError(t, err)
-	require.Equal(t, code, predeploys.EIP4788ContractCode)
+	require.Equal(t, predeploys.EIP4788ContractAddr, expected4788Address)
+	code, err := ethCl.CodeAt(context.Background(), expected4788Address, latestBlock.Number())
+	require.NoError(t, err)
+	require.NotEmpty(t, code)
+	require.Equal(t, predeploys.EIP4788ContractCode, code)
+	require.NotEmpty(t, predeploys.EIP4788ContractAddr)
+	require.Equal(t, predeploys.EIP4788ContractCodeHash, crypto.Keccak256Hash(code))
```

Comment:
Why not a `verfiyCodeHashMatches` here?

Reply:
I'll add the `NotEmpty` check to `verfiyCodeHashMatches`, good idea. And yes, will change it to use fixed-size type, no risk of accidental empty code hash slice then.